lib/attr_encrypted/adapters/active_record.rb

```diff
@@ -62,7 +62,7 @@ def attr_encrypted(*attrs)
 
             if ::ActiveRecord::VERSION::STRING >= "4.1"
               define_method("#{attr}_changed?") do |options = {}|
-                attribute_changed?(attr, options)
+                attribute_changed?(attr, **options)
```

Comment:
LGTM, doesn't seem to break ruby 2.7.5 compatibility
![image](https://user-images.githubusercontent.com/1923678/165829146-5d36462e-2c51-46d4-8c28-b8945d9ee4c5.png)